### ISSUE TITLE
XYZ plot don't change FLUX chekpoints

### ISSUE THREAD

**Issue by Desm0nt** [lllyasviel/stable-diffusion-webui-forge#1235]
If I select chekpoints as field for XYZ plot - it do nothin and generates all images with last (current) chekpoint, so we can't compare different Flux quants in a grid.

**Comment by l0stl0rd** [comment]
same issue here

**Comment by MBurneracc** [comment]
XYZ plots don't work for ANY checkpoints and haven't for like two weeks now, guy that reported it originally even got tired of waiting and found the cause themselves was Illya arbitrarily deleting the necessary code during the construction.  Fixing what they broke doesn't seem to be a priority.

**Comment by Auryg** [comment]
> XYZ plots don't work for ANY checkpoints and haven't for like two weeks now, guy that reported it originally even got tired of waiting and found the cause themselves was Illya arbitrarily deleting the necessary code during the construction. Fixing what they broke doesn't seem to be a priority.

If it's that big of a deal for you revert to a previous commit.  lllya has been putting in a ton of work and it'll probably be easier for him to go back through and fix bugs once everything is in place rather than tiptoeing around the code.

**Comment by kerizane** [comment]
Duplicate of https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/952

**Comment by Asaghon** [comment]
Anyone know at which commit this still works? It's the main reason I use Forge for :p
